### PR TITLE
about-us section is added

### PR DESCRIPTION
**Pull request by jshah436**

### CODE REVIEW

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-import { AddTutorialComponent } from './components/add-tutorial/add-tutorial.component';
+
 
 const routes: Routes = [
-  { path: '', redirectTo: 'tutorials', pathMatch: 'full' },
-  { path: 'tutorials', component: TutorialsListComponent },
-  { path: 'tutorials/:id', component: TutorialDetailsComponent },
-  { path: 'add', component: AddTutorialComponent }
+  { path: '', redirectTo: 'tutorials', pathMatch: 'full' }
 ];
```
Comment (Mushaf-Cyber) [inline spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts:8]: you haven't added routing for about us page?

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/about-us/about-us.component.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      </div>
+    </div>
+    <div class="get-a-quote-parent">
+      <div class="get-a-quote">Get a Quote</div>
+      <div class="design-your-home">Design your home today</div>
+      <button class="button">
+        <button class="content">
+          <div class="bg">
+          <div class="learn-more">Appointment 
+          </div>
```
Comment (Mushaf-Cyber) [inline spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/about-us/about-us.component.html:51]: have you added CSS i didn't see any CSS in css file

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts (hunk 1 above)
Comment (jshah436) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts:8]: routing file?

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/about-us/about-us.component.html (hunk 2 above)
Comment (jshah436) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/about-us/about-us.component.html:51]: added.

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts (hunk 1 above)
Comment (Mushaf-Cyber) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts:11]: routing for about in this file

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
-import { TutorialDetailsComponent } from './components/tutorial-details/tutorial-details.component';
-import { AddTutorialComponent } from './components/add-tutorial/add-tutorial.component';
-
+import { AboutUsComponent } from './components/about-us/about-us.component';
 const routes: Routes = [
-  { path: '', redirectTo: 'tutorials', pathMatch: 'full' },
-  { path: 'tutorials', component: TutorialsListComponent },
-  { path: 'tutorials/:id', component: TutorialDetailsComponent },
-  { path: 'add', component: AddTutorialComponent }
+  { path: '', redirectTo: 'tutorials', pathMatch: 'full'},
```
Comment (Mushaf-Cyber) [inline spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts:5]: please remove this routing

Comment (jshah436) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/app-routing.module.ts]: done

**Review by Mushaf-Cyber — APPROVED** [review]
approved

**Pull request merged**